Server/mods/deathmatch/logic/lua/CLuaFunctionDefs.Server.cpp

```diff
@@ -686,6 +686,25 @@ int CLuaFunctionDefs::SetServerConfigSetting(lua_State* luaVM)
     return 1;
 }
 
+int CLuaFunctionDefs::GetServerEnvironmentValue(lua_State* luaVM)
+{
+    SString strName;
+
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadString(strName);
+
+    if (!argStream.HasErrors())
+    {
+        lua_pushstring(luaVM, getenv(strName));
+        return 1;
+    }
+    else
+        m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());
+
+    lua_pushboolean(luaVM, false);
+    return 1;
+}
+
```

Comment:
```cpp
std::string_view CLuaFunctionDefs::GetServerEnvironmentValue(std::string name) 
{
    return getenv(name.c_str());
|
```
One thing im not sure about: What if `getenv` returns null? Does Lua handle it correctly?